packages/@ama-sdk/generator-sdk/src/generators/shell/templates/base/package.json

```diff
@@ -84,7 +84,7 @@
     "@o3r/eslint-plugin": "^<%= sdkCoreVersion %>",
     "@swc/cli": "^0.1.57",
     "@swc/core": "^1.2.249",
-    "@types/jasmine": "~4.3.5",
+    "@types/jasmine": "4.3.5",
```

Comment:
Why? (and why only in this package?)

Reply:
I think it's the version harmoniser, I wll put the `~` back everywhere.